classes/script_metadata.php

```diff
@@ -209,18 +209,38 @@ public static function get_redactable_param_names(): array {
      *
      * @return string the request path for this profile.
      */
-    public static function get_request(): string {
-        global $SCRIPT, $ME, $CFG;
+    public static function get_normalised_relative_script_path(): string {
```

Comment:
is there a reason to rename it?

Reply:
#285 renames it. I do think the name should be more descriptive and I am open to suggestions.

Reply:
ok cool new name is good